README.md

```diff
@@ -25,7 +25,7 @@ To get a local copy up and running follow these simple example steps.
 
```

Comment:
- [ ] It would be better if we add the video presentation about the project to our PR, this comment was raised by the previous code reviewer, hence try and fix it before submitting another code review

![Screenshot from 2022-08-03 14-30-42](https://user-images.githubusercontent.com/46388113/182634638-fe2e9f17-0647-4d49-82bd-cf8c4df39384.png)